### ISSUE TITLE
Support state attributes in condition like media player source

### ISSUE THREAD

**Issue by TripleEmcoder** [ericmatte/ad-media-lights-sync#57]
I would like to set up multiple sync pairs for the same media plyer (Spotify) to different lights (Living Room, Bedroom). The idea is that depending on the selected Spotify speaker lights sync in the appropriate room. The speaker is tracker in the `source` attribute of media player's state, so it is accessible in HA. However, the condition parameter of media_lights_sync only seems to support simple values. A way to drill down to state attributes would be a great addition.

**Comment by TripleEmcoder** [comment]
For anyone looking for a solution here's a way to expose an attribute as state with automatic variables: https://github.com/snarky-snark/home-assistant-variables#automatic-updates

**Comment by ericmatte** [comment]
This is a good idea! The `condition` attribute is used to know if the app should sync the lights or not.
This could be done like this:
```yaml
  condition:
    entity: input_boolean.sync_tv_lights
    state: "on"
    attributes:
        a_value: "low"
        another_value: "high"
```

And if all the given attributes match the selected condition entity, then it would sync.

Would that make sense with you?

**Comment by TripleEmcoder** [comment]
Yes, that's exactly what I meant. However, since then I discovered I also want sync to additionaly depend on the state of a different entity (`sun.sun` to be precise), so setting up a variable that merges all the state and attributes needed was the right solution.

**Comment by ericmatte** [comment]
All right 👍 Maybe I could at least update the documentation here to provide examples of more complexe conditions using HA automations or the link you've provided.

**Comment by ericmatte** [comment]
@TripleEmcoder I've just released [v1.2.0](https://github.com/ericmatte/ad-media-lights-sync/releases/tag/v1.2.0) which includes a way to use template value as condition directly in the configuration file.

Cheers
